src/modules/configurator/components/ConfigPropertyView.tsx

```diff
@@ -0,0 +1,31 @@
+import { ConfigProperty } from "./ConfigProperty";
+import { CarProperty } from "../types";
+
+export function ConfigPropertyView({
+  index,
+  propertyName,
+  modelName,
+  label,
+  name,
+  description,
+  price,
+}: CarProperty) {
+  return (
+    <div className="flex items-center justify-between cursor-text">
+      <ConfigProperty
+        index={index}
```

Comment:
You could spread the props here and than later just use props.price

Reply:
Spread props and used props.price.